cnd/src/btsieve/bitcoin/serde_bitcoind_network.rs

```diff
@@ -0,0 +1,30 @@
+use serde::{de, export::fmt, Deserializer};
+
+pub fn deserialize<'de, D>(deserializer: D) -> Result<bitcoin::Network, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    struct Visitor;
+
+    impl<'de> de::Visitor<'de> for Visitor {
+        type Value = bitcoin::Network;
+
+        fn expecting(&self, formatter: &mut fmt::Formatter<'_>) -> fmt::Result {
+            formatter.write_str("a bitcoin network")
+        }
+
+        fn visit_str<E>(self, v: &str) -> Result<Self::Value, E>
+        where
+            E: de::Error,
+        {
+            match v {
+                "main" => Ok(bitcoin::Network::Bitcoin),
```

Comment:
This means we are incompatible with older versions now, can we maintain backwards compatibility here?

Reply:
I checked with 0.17. and it was also `main`, `test` , and `regtest`.

Reply:
Oh I didn't know that, fine with me then :)